src/server/getters/get-reporting-fees.ts

```diff
@@ -12,6 +12,7 @@ export interface FeeDetails {
     unclaimedEth: string;
     unclaimedRepStaked: string;
     unclaimedRepEarned: string;
+    lostRep: string;
```

Comment:
So is this all REP that a user staked on an outcome that ultimately didn't become the final outcome?